wasm/dotnet/build.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+# Expects to have .NET SDK 9.0.3xx,
+# downloadable from using https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-win-x64.zip or https://aka.ms/dotnet/9.0.3xx/daily/dotnet-sdk-linux-x64.tar.gz
+
+dotnet workload install wasm-tools
+dotnet publish -o ./build-interp ./src/dotnet/dotnet.csproj
+dotnet publish -o ./build-aot ./src/dotnet/dotnet.csproj -p:RunAOTCompilation=true
```

Comment:
If I run this comment (and have cleared the `build-aot/` output from before), I don't see a `dotnet.native.js.symbols` file being created. @maraf How can I rebuild from source with off to the side symbols?

Reply:
Copy statements added to build.sh. Currently the .NET SDK doesn't support publishing symbols to the output

Reply:
In other words, to produce the symbols file, I need to build on Windows with Visual Studio? Or is there some other way to rebuild via the command-line on Linux or Mac OS?